backend/src/test/java/gov/cdc/usds/simplereport/api/BaseApiTest.java

```diff
@@ -144,6 +118,7 @@ public void cleanup() {
    */
   protected ObjectNode runQuery(String queryFileName) {
     try {
+      setTemplateAuthorization();
```

Comment:
i'm a bit worried that the `use____()` methods above are relying on a subsequent call to `setTemplateAuthorization()` in order to work. seems like this could break easily if a query/mutation is run through another method, as it is in `executeAddPersonMutation()`. could we just call `setTemplateAuthorization()` in the `use____()` methods above?

Reply:
I went back and forth on that: eventually, I decided that I'd rather make `_template` private and refactor away the direct accesses from subclasses. No very specific reason, but potentially clearing the headers multiple times seemed wrong and possibly productive of future bugs (though I guess this also produces possible future bugs by clearing the headers right before sending the request...).

Could do both, of course! But for now, went with the access restriction.

Reply:
yeah, i'm worried about those "possible future bugs", perhaps at the very least add a comment in one or two places so a future dev doesn't understand why their custom-wired query run with `useOrgEntryOnly()` isn't doing what they'd expect?